### PR TITLE
Proposal Submission P2

### PR DESCRIPTION
**Pull request by foleyb25**

### CODE REVIEW

File: P2-Proposals/JewlzandtheBoyz-p2-proposal.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Jewlz-TheBoyz-EmailCategorizationBot-P2
+
+# Introduction
+For our P2, we have decided to make an automation in UI Path Studio based off of one of the suggestions in the RevaturePro Slides. We will be making a Email Categorization Bot.
+
+# Automation
+This bot will automatically sort and categorize the users emails by a certain set of rules. We plan on adding facial recogniztion, the ability to login to different email services, and anything else we have time for.
+
+# Flow Chart
```
Comment (kaurrevature) [inline P2-Proposals/JewlzandtheBoyz-p2-proposal.md:9]: Please embed image into the markdown

Comment (mariellenolasco) [reply P2-Proposals/JewlzandtheBoyz-p2-proposal.md:9]: I can't see the image

Comment (foleyb25) [reply P2-Proposals/JewlzandtheBoyz-p2-proposal.md:9]: So in the markdown we reference the image in the same directory as the markdown. Can we just upload our image to the directory where the markdown will be?

**Review by mariellenolasco — APPROVED** [review]
Proposal Accepted

**Pull request merged**